benchmark/benchmark.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# CPU Solvers
+for i in `ls case*` 
+# for i in case300.m 
+do
+  for j in DirectSolver
+  do
+    julia --project=.. benchmarks.jl $j CPU $i | tail -1 >> cpu.log
```

Comment:
Could we move `julia` executable as a bash variable? That would allow to easily change the Julia version we are using, to benchmark different versions of Julia together :-)

Reply:
I'm not sure since there is no standard environment variable for the Julia executable. This assumes that the environment takes care of that when running the script.

Reply:
I was just thinking of setting a local variable at the beginning of the script: 
```bash
JULIA_BIN=julia

for j in DirectSolver
  do
    $JULIA_BIN --project=.. benchmarks.jl $j CPU $i | tail -1 >> cpu.log
```

Reply:
Ok added. Not sure what it changes though.